src/classes/CMT_MetadataAPI.cls

```diff
@@ -163,7 +163,7 @@ public class CMT_MetadataAPI {
                 upsert crlpSettings;
             }
             if (isSuccess) {
-                UTIL_OrgTelemetry_SVC.submitFeatureTelemetryToLMO();
+                UTIL_OrgTelemetry_SVC.processNPSPSettingTelemetry_ASync();
```

Comment:
Would you prefer that as `processNPSPSettingTelemetry_Async()`?

Reply:
That one or renaming it just as `asyncProcessNpspSettings()`. We already know it is telemetry since it is used in the `UTIL_OrgTelemetry_SVC`, but using `asyncProcessNpspSettingsTelemetry()` is fine too.

Reply:
Agreed. As a habit, I've been using the `_Async` suffix on my asynchronous methods to differentiate them from non-async methods of the same name. That works nicely in cases where I had to have two different entry points for an action. However, that's definitely not the case here, so I'll change the method name.